### PR TITLE
Send a guild message when a player reaches a level divisible by 10

### PR DESCRIPTION
**Pull request by BigSchnozz1**
Only did a brief test with the divisible by 10 condition removed, need to give it a go properly

### CODE REVIEW

**Review by blenderer — COMMENTED** [review]
This is wholesome. @BigSchnozz1 can you show a screenshot of this working?

File: Hardcore.lua [hunk 1]
```diff
@@ -263,6 +263,17 @@ function Hardcore:PLAYER_LEVEL_UP(...)
 	--take screenshot (got this idea from DingPics addon)
 	-- wait a bit so the yellow animation appears
 	C_Timer.After(PICTURE_DELAY, Screenshot)
+
+	-- send a message to the guild if the player's level is divisible by 10
+	local landmarkLevel = (level % 10) == 0
+	if (landmarkLevel) then
+		local playerName = UnitName("player")
+		local localizedClass = UnitClass("player")
```
Comment (RobertB4) [inline Hardcore.lua:271]: This is kind of a nit, but if someone is running e.g. the German or French client, the message sent to guild chat would be a bit weird having a non-English word in an English sentence.

According to the docs `UnitClass` also returns the english class name:
https://wowwiki-archive.fandom.com/wiki/API_UnitClass

I would change it to:

```
local _, englishClass = UnitClass("player")
```

Comment (BigSchnozz1) [reply Hardcore.lua]: Done, cheers

Comment (BigSchnozz1) [reply Hardcore.lua]: This causes allcaps, weirdly.

We use localised classes in other messages, so I'm going to revert this.

**Comment by BigSchnozz1** [comment]
Untitled

**Comment by BigSchnozz1** [comment]
> 
> 
> Untitled

This is in allcaps because of the change to use the ENG version of the classname instead of localised, so I'll revert that commit

**Comment by zdeyn** [comment]
Since all character names start with a single uppercase letter, and the rest are always lowercase - we could infer the name for the name that comes back - if not, we can use a simple lookup table to convert to the correct way to display that class.. it's not hard, but it's time - the feature is nice, and even with ALLCAPS for a class name, I think we can fit it in

Does it WORK, even if it looks naff?

**Review by zdeyn — APPROVED** [review]
If it looks shitty, but works? Ship it - can patch the text/visuals within a few days

**Comment by clawclasw** [comment]
> Since all character names start with a single uppercase letter, and the rest are always lowercase - we could infer the name for the name that comes back - if not, we can use a simple lookup table to convert to the correct way to display that class.. it's not hard, but it's time - the feature is nice, and even with ALLCAPS for a class name, I think we can fit it in
> 
> Does it WORK, even if it looks naff?

It worked with a test for levelup from 9-10, which is where the screenshot came from. I've changed the classname to use the localised version, to match the guildchat death messages, so the classname is no longer in allcaps. :+1:

**Review by 1010101110 — APPROVED** [review]

**Review by blenderer — APPROVED** [review]

**Pull request merged**